sentry-compose-helper/src/jvmMain/java/io/sentry/compose/gestures/ComposeGestureTargetLocator.java

```diff
@@ -19,6 +20,10 @@
 @SuppressWarnings("KotlinInternalInJava")
 public final class ComposeGestureTargetLocator implements GestureTargetLocator {
 
+  public ComposeGestureTargetLocator() {
+    SentryIntegrationPackageStorage.getInstance().addIntegration("ComposeHelper");
```

Comment:
```suggestion
    SentryIntegrationPackageStorage.getInstance().addIntegration("ComposeUserInteraction");
    SentryIntegrationPackageStorage.getInstance().addPackage("maven:io.sentry:sentry-compose", BuildConfig.VERSION_NAME)
```
since it's embedded in `sentry-compose`